buildenv

```diff
@@ -33,29 +20,63 @@ export ZOPEN_CLEAN="zopen_clean"
 zopen_init()
 {
   export CGO_ENABLED=0
-  # Go installs binaries, so create the bin dir as well
-  export GOBIN=$ZOPEN_INSTALL_DIR/bin
+  export GOTMPDIR=$PWD/go-tmpdir
+  mkdir -p $PWD/go-tmpdir
   export PATH=$PATH:$GOROOT/go-build-zos/bin
-  mkdir -p $ZOPEN_INSTALL_DIR
+  export GOBIN=$ZOPEN_INSTALL_DIR/bin
+  mkdir -p $ZOPEN_INSTALL_DIR/bin
+  unset CC CXX
 }
 
-zopen_wharf()
+zopen_config()
 {
-  cd ..
-
-  go work init ./terraform
-  wharf ./terraform/...
-  cd ./terraform
+  GOUSERDIRS_TAG=$(cat go.mod | grep go-userdirs | cut -d '-' -f 4 | cut -c 1-7)
+  READLINE_TAG='main' # $(cat go.mod | grep readline | cut -d '-' -f 3 | cut -c 1-7)
+  AFERO_TAG=$(cat go.mod | grep afero | cut -d ' ' -f 2)
+  # SPEAKEASY_TAG='main' # $(cat go.mod | grep speakeasy | cut -d ' ' -f 2)
+
+  cd .. && echo ""
+  git clone https://github.com/apparentlymart/go-userdirs.git
+  echo "Checking out $GOUSERDIRS_TAG"
+  cd go-userdirs && git -c advice.detachedHead=false checkout $GOUSERDIRS_TAG
+  # curl -s -o go-userdirs--$GOUSERDIRS_TAG.patch "https://raw.githubusercontent.com/ZOSOpenTools/wharf/main/deps-patches/go-userdirs--$GOUSERDIRS_TAG.patch"
+  # git apply -v --ignore-space-change --ignore-whitespace go-userdirs--$GOUSERDIRS_TAG.patch
+  git apply -v --ignore-space-change --ignore-whitespace ../deps-patches/go-userdirs--$GOUSERDIRS_TAG.patch
+
+  cd .. && echo ""
+  git clone https://github.com/chzyer/readline.git
+  echo "Checking out $READLINE_TAG"
+  cd readline
+  # curl -s -o readline--$READLINE_TAG.patch "https://raw.githubusercontent.com/ZOSOpenTools/wharf/main/deps-patches/readline--$READLINE_TAG.patch"
+  # git apply -v --ignore-space-change --ignore-whitespace readline--$READLINE_TAG.patch
+  git apply -v --ignore-space-change --ignore-whitespace ../deps-patches/readline--$READLINE_TAG.patch
+
+  cd .. && echo ""
+  git clone -c advice.detachedHead=false --branch $AFERO_TAG https://github.com/spf13/afero.git
+  echo "Checking out $AFERO_TAG"
+  cd afero
+  # curl -s -o afero--$AFERO_TAG.patch "https://raw.githubusercontent.com/ZOSOpenTools/wharf/main/deps-patches/afero--$AFERO_TAG.patch"
+  # git apply -v --ignore-space-change --ignore-whitespace afero--$AFERO_TAG.patch
+  git apply -v --ignore-space-change --ignore-whitespace ../deps-patches/afero--$AFERO_TAG.patch
+
+  # cd .. && echo ""
+  # git clone https://github.com/bgentry/speakeasy.git
```

Comment:
@v1gnesh I think, we can remove speakeasy clone since it's handled in go.mod.patch file

Reply:
Yup, removed the comments now